gen/dcos-config.yaml

```diff
@@ -284,17 +284,27 @@ package:
       OAUTH_ISSUER_URL={{ oauth_issuer_url }}
       OAUTH_CLIENT_ID={{ oauth_client_id }}
       OAUTH_AUTH_REDIRECTOR={{ oauth_auth_redirector }}
+  - path: /etc/lb_dnsnames
+    content: |
+      MASTER_INTERNAL_LB_DNSNAME={{ exhibitor_address }}
 {% switch master_discovery %}
 {% case "master_http_loadbalancer" %}
+  - path: /etc_master/dns_config_master
+    content: |
+      MASTER_SOURCE=exhibitor_uri
```

Comment:
I think we're good.

https://github.com/dcos/spartan/blob/master/src/spartan_config_loader_server.erl#L114